toolkit/primitives/domain/src/lib.rs

```diff
@@ -476,6 +476,33 @@ impl UtxoInfo {
 	}
 }
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+#[cfg_attr(feature = "clap", derive(clap::ValueEnum))]
```

Comment:
Where do you expect user to pass this parameter?
Isn't our code smart enough to get it from Ogmios?